src/Fhir.Anonymizer.R4.CommandLineTool/configuration-sample.json

```diff
@@ -1,200 +1,771 @@
 {
+  "fhirVersion": "R4",
   "fhirPathRules": [
-    {"path": "nodesByType('Extension')", "method": "redact"},
```

Comment:
While adding "R4", we should make the rules in one line to shrink the config size